src/main/webapp/data/developers.json

```diff
@@ -1556,6 +1559,14 @@
       "multiple": true,
       "name": "Zhu Liang",
       "username": "paradite"
+    },
+    {
+      "name": "ah91086",
+      "username": "ah91086"
+    },
+    {
+      "name": "cybersol795",
```

Comment:
If the actual name cannot be found, leave the `name` field empty.

Reply:
Got it :)